src/app/Shared/Services/NotificationChannel.service.tsx

```diff
@@ -93,24 +92,24 @@ export class NotificationChannel {
       notifications.success('Recording Created', `${event.recording} created in target: ${event.target}`);
     });
 
-    this.messages(NotificationCategory.RecordingSaved).subscribe(v => {
+    this.messages(NotificationCategory.RecordingStopped).subscribe(v => {
       const event: RecordingNotificationEvent = v.message;
-      notifications.success('Recording Archived', `${event.recording} was archived`);
+      notifications.success('Recording Stopped', `${event.recording} was stopped`);
     });
 
-    this.messages(NotificationCategory.RecordingArchived).subscribe(v => {
+    this.messages(NotificationCategory.RecordingSaved).subscribe(v => {
       const event: RecordingNotificationEvent = v.message;
-      notifications.success('Recording Archived', `${event.recording} was archived`);
+      notifications.success('Recording Archived', `${event.recording.name} was uploaded into archives`);
     });
 
-    this.messages(NotificationCategory.RecordingStopped).subscribe(v => {
+    this.messages(NotificationCategory.RecordingArchived).subscribe(v => {
       const event: RecordingNotificationEvent = v.message;
-      notifications.success('Recording Stopped', `${event.recording} was stopped`);
+      notifications.success('Recording Archived', `${event.recording.name} was archived`);
     });
 
     this.messages(NotificationCategory.RecordingDeleted).subscribe(v => {
       const event: RecordingNotificationEvent = v.message;
-      notifications.success('Recording Deleted', `${event.recording} was deleted`);
+      notifications.success('Recording Deleted', `${event.recording.name} was deleted`);
```

Comment:
Deleting archived recordings displays the correct file name in the notification, but when I delete a running/stopped recording, the notification reads `undefined was deleted`.

Reply:
fixed

Reply:
Deleting the active recordings is working well now. When I tried to delete an archived recording, I got a notification reading `[object Object] was deleted`.

Reply:
Ah. Turns out that deleting an active or an archived recording would each trigger a notification with the same category, but (for now - see also #368) different message contents format. I have updated this PR and the corresponding backend one to use more specific categories for these notifications, and to disambiguate the archived/active deletion.